### PR TITLE
Docs: incorporate architect feedback

### PR DESCRIPTION
**Pull request by tom-rm-meyer-ISST**
## Description

I received first feedback from system architects upfront the QGate, that I incorporated:
- more visualization in the user guide
- linkage from admin guide to interface doc

Beside that I updated non-rendering images.

As we meanwhile implemented the data sovereignty concept, I also added documentation to handle to following "remember to do it" issues:
- solves #239 
- solves #240 

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: docs/arc42/about-arc42.md
Comment (mhellmeier) [inline docs/arc42/about-arc42.md:1]: Are you sure that this file should be included in the repository?

File: docs/arc42/arc42-template.md
Comment (mhellmeier) [inline docs/arc42/arc42-template.md:1]: Are you sure that this file should be included in the repository?

File: docs/arc42/about-arc42.md
Comment (tom-rm-meyer-ISST) [reply docs/arc42/about-arc42.md:1]: You're right, that has been included without intention. Removed

File: docs/arc42/arc42-template.md
Comment (tom-rm-meyer-ISST) [reply docs/arc42/arc42-template.md:1]: You're right, that has been included without intention. Removed

**Review by mhellmeier — APPROVED** [review]

**Pull request merged**